lbrynet/lbrynet_daemon/UIManager.py

```diff
@@ -95,6 +94,8 @@ def setup(self, branch=None, check_requirements=None, user_specified=None):
 
         d = self._up_to_date()
         d.addCallback(lambda r: self._download_ui() if not r else self._load_ui())
+        if launch:
+            d.addCallback(lambda _: webbrowser.open(conf.settings.UI_ADDRESS))
```

Comment:
On line 75, 83, 89, the function returns which means launch won't happen regardless of the setting.

In general, functions should "do one thing" and now this function does two things.  It setups the UI and potentially launches it.

Also - I've harped on boolean flags before: http://www.informit.com/articles/article.aspx?p=1392524

I think it would be better to have this be two separate functions and call them individually.